src/main/java/http/logger/Logger.java

```diff
@@ -0,0 +1,22 @@
+package http.logger;
+
+import http.request.HttpRequest;
+
+public class Logger implements ILogger {
+    private StringBuilder log;
+
+    public Logger() {
+        this.log = new StringBuilder();
+    }
+
+    @Override
+    public void addRequest(HttpRequest httpRequest) {
```

Comment:
its kind of weird that the logger knows about HttpRequest. can this method instead just accept a string instead? renaming this to something more generic like 'info' would also mean it can be used for other logging.

Reply:
👍 makes sense